### PR TITLE
bump opencontainers/selinux to v1.2

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @runcom @justincormack @vdemeester PTAL

File: vendor/github.com/opencontainers/selinux/go-selinux/label/label_selinux.go [hunk 1]
```diff
@@ -90,14 +104,36 @@ func SetProcessLabel(processLabel string) error {
 	return selinux.SetExecLabel(processLabel)
 }
 
+// SetSocketLabel takes a process label and tells the kernel to assign the
+// label to the next socket that gets created
+func SetSocketLabel(processLabel string) error {
```
Comment (thaJeztah) [inline vendor/github.com/opencontainers/selinux/go-selinux/label/label_selinux.go]: @runcom Is this something we need to use for `/var/run/docker.sock` (and/or use in containerd to set labels on sockets?)

File: vendor/github.com/opencontainers/selinux/go-selinux/label/label_selinux.go [hunk 2]
```diff
@@ -130,13 +166,56 @@ func Relabel(path string, fileLabel string, shared bool) error {
 		return nil
 	}
 
-	exclude_paths := map[string]bool{"/": true, "/usr": true, "/etc": true, "/tmp": true, "/home": true, "/run": true, "/var": true, "/root": true}
+	exclude_paths := map[string]bool{
```
Comment (thaJeztah) [inline vendor/github.com/opencontainers/selinux/go-selinux/label/label_selinux.go]: pls double-check if the new list will cause any changes in behavior that we should be aware of 🤗

**Comment by thaJeztah** [comment]
ping @justincormack @runcom PTAL 🤗

**Review by justincormack — APPROVED** [review]

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Pull request merged**